dmoj/executors/NETCS.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from dmoj.cptbox.filesystem_policies import ExactFile, RecursiveDir
+from dmoj.executors.compiled_executor import CompiledExecutor
+
+CSPROJ = b"""\
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PublishSingleFile>true</PublishSingleFile>
+  </PropertyGroup>
+</Project>
+"""
+
+HELLO_WORLD_PROGRAM = """\
+Console.WriteLine("Hello, World!");
+"""
```

Comment:
I know RUST does this, but since this is one line, and we don't have any of RUST's "prefetch libraries" business, let's remove this and inline it.